demo_waterfall.py

```diff
@@ -17,23 +17,24 @@
 
 from __future__ import division
 import matplotlib.animation as animation
-from matplotlib.mlab import psd
-import pylab as pyl
+import matplotlib.pyplot as plt
 import numpy as np
 import sys
 from rtlsdr import RtlSdr
+from scipy.signal import welch
+from numba import jit
```

Comment:
Can this be optional?  I'd rather not add any dependencies than necessary.

Maybe if you move the related [log10](https://github.com/pyrtlsdr/pyrtlsdr/pull/165/files#diff-f0bc4f71dc1f49e0f3ddf82b5fa4d57e1152a910d2747f53a41aa1832109acc9R129-R135) function up here and wrap it all in a

```python
try:
    from numba import jit

    @jit(...)
    def log10(...):
        ...

except ImportError:
    log10 = np.log10
```